### PR TITLE
Add manifest for org.kde.krdc

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.krdc

**Comment by flyingcakes85** [comment]
@travier 

I have tested VNC to work :+1: Couldn't get RDP to work, which is _probably_ because of my inexperience with the Windows environment.

Also, should we need pulseaudio sandbox permission here? I don't get any error messages in terminal when using VNC, but I believe RDP also carries audio.

**Comment by flyingcakes85** [comment]
bot, build org.kde.krdc

File: org.kde.krdc.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.krdc.json:17]:
```suggestion
        "--socket=fallback-x11",
```

File: org.kde.krdc.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        "--socket=wayland",
+        "--share=network",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "libusb",
```
Comment (hfiguiere) [inline org.kde.krdc.json:24]: you should get it from shared-modules.

File: org.kde.krdc.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FreeRDP/FreeRDP/archive/refs/tags/2.5.0.tar.gz",
+                    "sha256": "1bf305141f9e0b9dce6b854ab761b5325b5a9d3b610de47e5c35edb49b3b4dcb"
+                }
+            ]
+        },
+        {
+            "name": "icon",
```
Comment (hfiguiere) [inline org.kde.krdc.json:69]: this should be part of the main module (`post-install` section)

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.krdc.json (hunk 2 above)
Comment (flyingcakes85) [reply org.kde.krdc.json]: Thanks, updated :+1:

**Comment by flyingcakes85** [comment]
bot, build org.kde.krdc

**Comment by travier** [comment]
CC @flathub/kde

**Comment by travier** [comment]
Pulseaudio should be fine

**Comment by travier** [comment]
Not yet tested but LGTM

**Comment by flyingcakes85** [comment]
bot, build org.kde.krdc

**Comment by flyingcakes85** [comment]
Added pulseaudio permission.

**Pull request closed without merging**